### DISCUSSION TITLE
Switch topic

### DISCUSSION THREAD

**Discussion by chuckyro** [arendst/Tasmota#17276] [Q&A]
Hi guys,

I have a small problem/question. 

I've configured a device and set up 2 switches (2xGPIOs).  I detached them for relays. Everything works fine!
But ..every time a change the status on one switch, tasmota sends a mqtt message with both statuses ! 
Is there a config to force the device to send only one new status (the affected switch) ?! 

eg: //not desired :(
/tele/SENSOR = {"Time":"2022-11-28T11:19:05","Switch1":"ON","Switch2":"OFF"}   // only Switch1 changed

eg: // desired :)
/tele/SENSOR = {"Time":"2022-11-28T11:19:05","Switch1":"ON"}     //Switch1 changed
/tele/SENSOR = {"Time":"2022-11-28T11:19:07","Switch2":"OFF"}    //Switch2 changed

Thanks.

D.

**Comment by barbudor** [comment]
I think you are using switchmode 15
May be try SetOption114 which may behave differently

**Reply by chuckyro** [reply]
negative !   "switchmode[x] 15 " send only MQTT message on switch change. I already did that.

**Comment by chuckyro** [comment]
so no hints here ?
